core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ApplicationSummaryInfo.scala

```diff
@@ -196,4 +196,32 @@ class SingleAppSummaryInfoProvider(val app: ApplicationSummaryInfo)
       0.0
     }
   }
+
+  /**
+   * Check if there are any scan stages with failed tasks due to GPU OOM errors
+   * (GpuRetryOOM and GpuSplitAndRetryOOM).
+   */
+  override def hasScanStagesWithGpuOom: Boolean = {
+    // Find stages with failed tasks due to GPU OOM errors
```

Comment:
I don't have strong recollection of the behavior. If ProfilingAutoTuner accepts CPU eventlogs, then this method should return false for non GPU eventlog.

Reply:
Added early check for non-GPU event logs.

Reply:
great!